tests/plugins/clipboard/dropImage.js

```diff
@@ -0,0 +1,169 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: clipboard */
+/* bender-include: _helpers/pasting.js */
+
+( function() {
+	'use strict';
+
+	var setHtmlWithSelection = bender.tools.setHtmlWithSelection,
+		onDrop,
+		onPaste;
```

Comment:
They can be moved to the beginning of `assertDropImage` function. IMO it would be more readable that way.

Reply:
I can't move this. I use this for removing listeners in `setUp` function.